tests/components/usgs_earthquakes_feed/test_geo_location.py

```diff
@@ -215,18 +239,16 @@ def _generate_mock_feed_entry(
 async def test_setup_with_custom_location(hass):
     """Test the setup with a custom location."""
     # Set up some mock feed entries for this test.
-    mock_entry_1 = _generate_mock_feed_entry("1234", "Title 1", 20.5, (-31.1, 150.1))
+    mock_entry_1 = _generate_mock_feed_entry("1234", "Title 1", 20.5,
+                                             (-31.1, 150.1))
 
-    with patch(
-        "geojson_client.usgs_earthquake_hazards_program_feed."
-        "UsgsEarthquakeHazardsProgramFeed"
-    ) as mock_feed:
+    with patch("geojson_client.usgs_earthquake_hazards_program_feed."
+               "UsgsEarthquakeHazardsProgramFeed") as mock_feed:
         mock_feed.return_value.update.return_value = "OK", [mock_entry_1]
 
         with assert_setup_component(1, geo_location.DOMAIN):
-            assert await async_setup_component(
-                hass, geo_location.DOMAIN, CONFIG_WITH_CUSTOM_LOCATION
-            )
+            assert await async_setup_component(hass, geo_location.DOMAIN,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4643140)